{
	"id": "AVG-693",
	"advisories": [
		{
			"content": {
				"id": "ASA-201805-10",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/ASA-201805-10"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "ASA-201805-10",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/ASA-201805-10"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "AVG-693",
				"severity": [
					{
						"type": "vendor",
						"source": "security.archlinux.org",
						"vendor": "Critical"
					}
				],
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/AVG-693"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2018-5150",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5150"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5151",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5151"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5152",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5152"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5153",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5153"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5154",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5154"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5155",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5155"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5157",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5157"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5158",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5158"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5159",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5159"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5160",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5160"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5163",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5163"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5164",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5164"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5166",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5166"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5167",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5167"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5168",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5168"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5169",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5169"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5172",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5172"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5173",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5173"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5175",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5175"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5176",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5176"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5177",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5177"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5180",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5180"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5181",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5181"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2018-5182",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-5182"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "arch",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"type": "binary",
										"binary": {
											"name": "firefox"
										}
									},
									"affected": {
										"type": "pacman",
										"range": [
											{
												"lt": "60.0-1"
											}
										],
										"fixed": [
											"60.0-1"
										]
									}
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "arch",
		"raws": [
			"vuls-data-raw-arch/AVG-693.json"
		]
	}
}
